pitest-maven/src/main/java/org/pitest/maven/PitMojo.java

```diff
@@ -47,7 +47,7 @@
    * @parameter expression="${targetClasses}"
    *
    */
-  protected ArrayList<String>         targetClasses;
+  private List<String>         targetClasses;
```

Comment:
I know and you already merged upgrade to maven 3, see #199 

So this should be not an issue.